core/templates/layout.user.php

```diff
@@ -69,6 +69,7 @@
 					<?php foreach ($_['navigation'] as $entry): ?>
 						<li data-id="<?php p($entry['id']); ?>" class="hidden" tabindex="-1">
 							<a href="<?php print_unescaped($entry['href']); ?>"
+								<?php if ($entry['target']): ?> target="_blank" rel="noreferrer noopener"<?php endif; ?>
```

Comment:
Not sure where $entry['target'] comes from, but can it be something else than `blank` ?

```suggestion
								<?php if ($entry['target'] === "blank"): ?> target="_blank" rel="noreferrer noopener"<?php endif; ?>
```

Reply:
It's new, so no.
Boolean sounds more sane to reduce issues?

https://github.com/nextcloud/external/pull/287

Reply:
Ah, thanks. If it's new then yes, let's go for boolean, but maybe with a more explicit name?

Reply:
`target_blank` would be enough